UPGRADE-7.1.md

```diff
@@ -0,0 +1,12 @@
+UPGRADE FROM 7.0 TO 7.1
```

Comment:
This file needs to be removed and the text needs to be added to the UPGRADE-7.0.md

Reply:
ok